app/views/branches/health.html.haml

```diff
@@ -26,6 +26,32 @@
       %span.number= rate
       %span.label pass rate on first try
 
+  .stats
+    %h2.subheader= "Build statistics"
+    %p= "Build created on #{@first_built_date.strftime("%Y-%m-%d")}"
```

Comment:
The `=` sign is not needed here and on many of the lines below. `#{}` interpolation can be used on plain `%tag` lines.

I didn't realize this about haml either for a long time.